android/build.gradle

```diff
@@ -13,20 +13,12 @@ android {
         targetSdkVersion safeExtGet('targetSdkVersion', 27)
         versionCode 1
         versionName "1.0"
-        ndk {
-            abiFilters "armeabi-v7a", "x86"
-        }
     }
-    splits {
-        abi {
-            reset()
-            enable safeExtGet('enableSeparateBuildPerCPUArchitecture', false)
-            universalApk safeExtGet('universalApk', false)  // If true, also generate a universal APK
-            include "armeabi-v7a", "x86"
-        }
+    lintOptions {
+        abortOnError false
     }
 }
 
 dependencies {
-    compileOnly "com.facebook.react:react-native:+"
+    implementation "com.facebook.react:react-native:+"
 }
```

Comment:
`create-react-native-library` has repos defined. Do we want to be explicit about those, or what does it default to?
```groovy
repositories {
  mavenCentral()
  google()
}


dependencies {
  // For < 0.71, this will be from the local maven repo
  // For > 0.71, this will be replaced by `com.facebook.react:react-android:$version` by react gradle plugin
  //noinspection GradleDynamicVersion
  implementation "com.facebook.react:react-native:+"
}
```

Reply:
Good catch. I'm not positive, but probably since we are pulling react native should probably match template

Reply:
Yep good call. Going to port over the property loading from the `gradle.properties` and clean that list down to just 3.